introduction/intro.md

```diff
@@ -13,166 +13,124 @@ redirect_from:
-are still **monolithic** in the sense that all software is still running in
-the same OS. This means, once again, that if your session is compromised,
-then all the data and activities performed within that same session are also
-potentially compromised.
-
-
-How does Qubes OS compare to running VMs in a conventional OS?
---------------------------------------------------------------
-
-Not all virtual machine software is equal when it comes to security. You may
-have used or heard of VMs in relation to software like VirtualBox or VMware
-Workstation. These are known as "Type 2" or "hosted" hypervisors. (The
-**hypervisor** is the software, firmware, or hardware that creates and
-runs virtual machines.) These programs are popular because they're designed
-primarily to be easy to use and run under popular OSes like Windows (which
-is called the **host** OS, since it "hosts" the VMs). However, the fact
-that Type 2 hypervisors run under the host OS means that they're really
-only as secure as the host OS itself. If the host OS is ever compromised,
-then any VMs it hosts are also effectively compromised.
-
-By contrast, Qubes uses a "Type 1" or "bare metal" hypervisor called
-[Xen]. Instead of running inside an OS, Type 1 hypervisors run directly on the
-"bare metal" of the hardware. This means that an attacker must be capable of
-subverting the hypervisor itself in order to compromise the entire system,
-which is vastly more difficult.
-
-Qubes makes it so that multiple VMs running under a Type 1 hypervisor can be
-securely used as an integrated OS. For example, it puts all of your application
-windows on the same desktop with special colored borders indicating the
-trust levels of their respective VMs. It also allows for things like secure
-copy/paste operations between VMs, securely copying and transferring files
-between VMs, and secure networking between VMs and the Internet.
-
-
-How does Qubes OS compare to using a separate physical machine?
----------------------------------------------------------------
-
-Using a separate physical computer for sensitive activities can certainly be
-more secure than using one computer with a conventional OS for everything,
-but there are still risks to consider. Briefly, here are some of the main
-pros and cons of this approach relative to Qubes:
-
-<div class="focus">
-  <i class="fa fa-check"></i> <strong>Pros</strong>
+  <div class="row">
+      <div class="col-lg-3 col-md-3 text-left">
+          <p>Qubes OS is a free and open-source security-oriented operating system meant for single-user desktop computing.</p> 
+          <p>Qubes OS leverages <a href="https://wiki.xen.org/wiki/Xen_Project_Software_Overview/">xen-based virtualization</a> to allow for the creation and management of isolated virtual machines called <a href="/doc/glossary#qube">qubes</a>. 
+          Qubes, which are also referred to as <a href="/doc/glossary#qube/">domains</a> or compartments, have specific :</p>
```

Comment:
This link is broken, it should be `/doc/glosary/#qube` instead.

Reply:
Sorry for the error and thank you for reviewing my pull request. I have corrected the broken link in patch-4.